### ISSUE TITLE
v31.0.0 - npm install fails

### ISSUE THREAD

**Issue by puzrin** [rxaviers/cldr-data-npm#43]
I think something is broken on this release.

```sh
npm install cldr-data
```

node v6, happens both on travis and locally.

```
npm install cldr-data
npm WARN deprecated npmconf@2.0.9: this package has been reintegrated into npm and is now out of date with respect to npm
npm WARN deprecated node-uuid@1.4.7: use uuid module instead

> cldr-data@31.0.0 install /home/vitaly/Загрузки/node_modules/cldr-data
> node install.js

GET `https://github.com/unicode-cldr/cldr-core/archive/31.0.0.zip`
GET `https://github.com/unicode-cldr/cldr-dates-modern/archive/31.0.0.zip`
GET `https://github.com/unicode-cldr/cldr-cal-buddhist-modern/archive/31.0.0.zip`
GET `https://github.com/unicode-cldr/cldr-cal-chinese-modern/archive/31.0.0.zip`
GET `https://github.com/unicode-cldr/cldr-cal-coptic-modern/archive/31.0.0.zip`
GET `https://github.com/unicode-cldr/cldr-cal-dangi-modern/archive/31.0.0.zip`
GET `https://github.com/unicode-cldr/cldr-cal-ethiopic-modern/archive/31.0.0.zip`
GET `https://github.com/unicode-cldr/cldr-cal-hebrew-modern/archive/31.0.0.zip`
GET `https://github.com/unicode-cldr/cldr-cal-indian-modern/archive/31.0.0.zip`
GET `https://github.com/unicode-cldr/cldr-cal-islamic-modern/archive/31.0.0.zip`
GET `https://github.com/unicode-cldr/cldr-cal-japanese-modern/archive/31.0.0.zip`
GET `https://github.com/unicode-cldr/cldr-cal-persian-modern/archive/31.0.0.zip`
GET `https://github.com/unicode-cldr/cldr-cal-roc-modern/archive/31.0.0.zip`
GET `https://github.com/unicode-cldr/cldr-localenames-modern/archive/31.0.0.zip`
GET `https://github.com/unicode-cldr/cldr-misc-modern/archive/31.0.0.zip`
GET `https://github.com/unicode-cldr/cldr-numbers-modern/archive/31.0.0.zip`
GET `https://github.com/unicode-cldr/cldr-segments-modern/archive/31.0.0.zip`
GET `https://github.com/unicode-cldr/cldr-units-modern/archive/31.0.0.zip`
/home/vitaly/Загрузки/node_modules/cldr-data-downloader/lib/download.js:66
      throw error;
      ^

Error: Error requesting archive.
Status: 404
Request options: {
  "url": "https://github.com/unicode-cldr/cldr-cal-chinese-modern/archive/31.0.0.zip",
  "headers": {
    "User-Agent": "curl/7.21.4 (universal-apple-darwin11.0) libcurl/7.21.4 OpenSSL/0.9.8r zlib/1.2.5"
  },
  "followRedirect": true,
  "encoding": null
}
Response headers: {
  "content-length": "15",
  "access-control-allow-origin": "https://render.githubusercontent.com",
  "content-security-policy": "default-src 'none'; style-src 'unsafe-inline'",
  "strict-transport-security": "max-age=31536000",
  "vary": "Authorization,Accept-Encoding",
  "x-content-type-options": "nosniff",
  "x-frame-options": "deny",
  "x-xss-protection": "1; mode=block",
  "date": "Mon, 20 Mar 2017 23:01:02 GMT",
  "x-github-request-id": "876D:2FE62:4569E3:596A79:58D05F2D",
  "connection": "close"
}
Make sure your network and proxy settings are correct.

If you continue to have issues, please report this full log at https://github.com/rxaviers/cldr-data-downloader
    at Request._callback (/home/vitaly/Загрузки/node_modules/cldr-data-downloader/lib/download.js:58:15)
    at Request.self.callback (/home/vitaly/Загрузки/node_modules/request/request.js:187:22)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request.<anonymous> (/home/vitaly/Загрузки/node_modules/request/request.js:1044:10)
    at emitOne (events.js:101:20)
    at Request.emit (events.js:188:7)
    at IncomingMessage.<anonymous> (/home/vitaly/Загрузки/node_modules/request/request.js:965:12)
    at emitNone (events.js:91:20)
    at IncomingMessage.emit (events.js:185:7)
npm WARN enoent ENOENT: no such file or directory, open '/home/vitaly/Загрузки/package.json'
```

**Comment by rxaviers** [comment]
Thanks for filing the issue. Unfortunately I pushed the tag too early.. It should work as soon as CLDR JSON publishes 31.0.0 as well in a couple of hours.

**Comment by rxaviers** [comment]
Fixed, CLDR JSON 31.0.0 is released.
